tools/libboard/board_common/board_common.c

```diff
@@ -500,5 +502,49 @@ fpga_result find_dev_feature(fpga_token token,
 	if (pglob.gl_pathv)
 		globfree(&pglob);
 
+	return res;
+}
+
+//Prints fpga boot page info
+fpga_result print_common_boot_info(fpga_token token)
+{
+	fpga_properties props       = NULL;
+	fpga_result res             = FPGA_OK;
+	fpga_result retval          = FPGA_OK;
+	uint64_t bbs_id             = (uint64_t)-1;
+	fpga_objtype objtype;
+
+	res = fpgaGetProperties(token, &props);
+	if (res != FPGA_OK) {
+		OPAE_ERR("Failed to get Properties ");
+		return res;
+	}
+
+	retval = fpgaPropertiesGetBBSID(props, &bbs_id);
+	if (retval != FPGA_OK) {
+		OPAE_ERR("Failed to get bbsid ");
+		res = retval;
+		goto pro_destroy;
+	}
+
+	retval = fpgaPropertiesGetObjectType(props, &objtype);
+	if (retval != FPGA_OK) {
+		OPAE_ERR("Failed to get object type ");
+		res = retval;
+		goto pro_destroy;
+	}
+
+	if (objtype == FPGA_DEVICE) {
+		printf("%-32s : %s\n", "Boot Page",
```

Comment:
We can only show the boot page for an FPGA_DEVICE?

Reply:
OPAE software gets boot page info from bitstream id for  N3000 & D5005 cards , bit 36 is 0 factory or 1 user 

OPAE  software gets boot page info from sysfs  fpga_boot_image for N61010 card.